src/Windows/Avalonia.Win32/DxgiSwapchain/Dxgi.idl

```diff
@@ -0,0 +1,305 @@
+﻿@clr-namespace Avalonia.Win32.DxgiSwapchain
```

Comment:
I think this file should be named `directx.idl` and live in `DirectX` directory so it could be extended with other DirectX-related APIs later.

Reply:
Same with DxgiEnums/DxgiStructs